src/semaphore/config.py

```diff
@@ -99,6 +100,11 @@ class Config(BaseSettings):
     )
     """URL for the redis instance, used by the worker queue."""
 
+    queue_name: str = Field("arq:queue", env="TS_REDIS_QUEUE_NAME")
+    """Name of the arq queue that the worker processes from."""
+
+    arq_mode: ArqMode = Field(ArqMode.production, env="TS_ARQ_MODE")
```

Comment:
I missed this in the last PR (see `redis_url`) but the env variable should be renamed so that it uses a `SEMAPHORE` prefix instead of `TS`.